sa/database.go

```diff
@@ -144,11 +144,8 @@ func initTables(dbMap *gorp.DbMap) {
 	regTable.ColMap("KeySHA256").SetNotNull(true).SetUnique(true)
 	pendingAuthzTable := dbMap.AddTableWithName(pendingauthzModel{}, "pending_authz").SetKeys(false, "ID")
 	pendingAuthzTable.SetVersionCol("LockCol")
-	pendingAuthzTable.ColMap("Challenges").SetMaxSize(1536)
-
-	authzTable := dbMap.AddTableWithName(authzModel{}, "authz").SetKeys(false, "ID")
-	authzTable.ColMap("Challenges").SetMaxSize(1536)
-
+	dbMap.AddTableWithName(authzModel{}, "authz").SetKeys(false, "ID")
+	dbMap.AddTableWithName(core.Challenge{}, "challenges").SetKeys(true, "ID").ColMap("Validation").SetMaxSize(1536)
```

Comment:
Don't we need some uniqueness requirements on these? Shouldn't it be one at most of each type per domain?